src/UI/ReplyActions.jsx

```diff
@@ -32,21 +71,20 @@ class ReplyActions extends Component {
 			this.sendMessage();
 		} else {
 			// Wait until device is subscribed before trying to send a message
-			ApiEventTarget.addEventListener(subscribe, this.handleSubscribe);
+			const handleSubscribe = (event) => {
+				if(!event.detail.errorNotifications)
+					this.sendMessage();
+
+				// This is a one time thing (for this submit),
+				// so stop listening for future subscriptions
+				ApiEventTarget.removeEventListener(subscribe, handleSubscribe);
+			};
```

Comment:
Waarom heb je de code hierheen verplaatst? En als het beter hier is, waarom dan niet meteen inline in de `addEventListener`

Reply:
Deze `handleSubmit` werkt bijna hetzelfde als `handleFileChange`, het enige verschil is dat de `handleSubscribe` van `handleFileChange` de `uploadMedia` aanroept (deze roept vervolgens `sendMedia` aan).

Dit verschilt omdat we media niet willen versturen met `sendMessage`, daar zit namelijk het disablen van het invoer veld en het legen van de reply tekst in. Dat moet niet gebeuren als je een media file upload (anders wordt je input gerest bij het uploaden)